src/Compatibility/Microsoft.DotNet.ApiCompatibility/ApiComparerFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
```

Comment:
Not exactly related to your change, but we should aim to remove this and instead annotate the whole library sooner rather than later. Having it partially like we do with this assembly is potentially dangerous due to the assumptions we make.

Reply:
Agreed. We have https://github.com/dotnet/sdk/issues/25760 to track that.